app/src/main/res/values/strings.xml

```diff
@@ -74,4 +74,7 @@
     <string name="title_add">Add</string>
     <string name="title_create">Create</string>
     <string name="action_x">x</string>
+    <string name="title_sales">Sales</string>
+    <string name="title_customers">Customers</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.title_customers` appears to be unused
 Unused resources make applications larger and slow down builds.